styles.css

```diff
@@ -0,0 +1,89 @@
+@import './components/vibeui.css';
+
+* {
+	margin: 0;
+	font-family: 'Roboto', sans-serif;
+	box-sizing: border-box;
+	word-wrap: break-word;
+}
+
+body{
+    position: relative;
+}
+
+.navbar{
+    padding: 1.5rem 3rem;
+    box-shadow: 0 8px 5px rgb(233, 230, 230);
+    display: flex;
+    justify-content: space-between;
+}
+.link{
+    text-decoration: none;
+    color: black;
+}
+.link:hover{
+    color: var(--primary-color);
+}
+.title{
+    display: inline;
+}
+
+.list{
+    list-style: none;
+    padding: 0;
+    font-size: 1.2rem;
+}
+
+.list-inline-item{
+    display: inline;
+    margin: 1rem;
+}
+.image{
+    width: 400px;
+}
+
+.image img{
+    width: 300px;
+}
+.row{
+    display: flex;
+    padding: 3rem 3rem 0 3rem;
+    align-items: center;
+    justify-content: center;
+    gap: 4rem;
+    flex-wrap: wrap;
+}
+.content{
+    width: 400px;
+    font-size: 1rem;
+    line-height: 1.5;
+}
+p{
```

Comment:
avoid using p tag directly

Reply:
yea, I will refactor this code once, I'm done with all the components. I will follow the best practices. Thanks